app/controllers/payment_controller.rb

```diff
@@ -1,32 +0,0 @@
-class PaymentController < ApplicationController
```

Comment:
This file was renamed from `payment_controller.rb` to `payments_controller.rb`. Rails was looking for the plural, couldn't find it and threw errors.